### PR TITLE
docs: added instruction from rpm to container

### LINKED ISSUE #1799: docs: add migration path from RPM to container
### Description

Right now there is no clear path in regards to how to migrate from an RPM installation to a containerized scenario. We should add a page on http://port.us.org talking about this.

### PR DESCRIPTION
**Pull request by mssola**
Fixes #1799

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: _docs/migrate-from-rpm.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Migrating from the RPM
+
+During the development cycle of the 2.3 release, we started to focus more and
+more on containerized deployments. The [official openSUSE Docker
+image](https://hub.docker.com/r/opensuse/portus/) got more attention and it
+started to be thinner and easier to deploy. Following current trends, we decided
+to make these kinds of deployments the preferred ones in the 2.3 release.
+
+The migration path from a pure RPM installation and a containerized one is not
```
Comment (vitoravelino) [inline _docs/migrate-from-rpm.md]: `to a` instead of `and a`

Comment (vitoravelino) [reply _docs/migrate-from-rpm.md]: I'd say to remove both `a` from `from` and `to`.

File: _docs/migrate-from-rpm.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+that big. That's because the Docker image simply installs the RPM as produced in
+our [OBS
+project](https://build.opensuse.org/project/show/Virtualization:containers:Portus:2.3). So
+from the distribution point of view (and the tooling) nothing changes: the
+"only" change is to go from a bare metal installation, to Docker containers.
```
Comment (vitoravelino) [inline _docs/migrate-from-rpm.md]: I guess comma is a typo

File: _docs/migrate-from-rpm.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Backing up
+
+First of all, you should proceed as you would with any upgrade: back up your
+data. There are two main things you should back up: images stored in the
+registry, and the database.
```
Comment (vitoravelino) [inline _docs/migrate-from-rpm.md]: I think comma could be removed.

**Comment by mssola** [comment]
@vitoravelino fixed everything except the part in which I was not sure.

**Pull request merged**